### PR TITLE
[11.0][MIG] res_config_settings_enterprise_remove: port to V11

### PR DESCRIPTION
**Pull request by astirpe**

### CODE REVIEW

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 1]
```diff
@@ -1,14 +1,58 @@
...
+    'module_inter_company_rules',
+    'module_mrp_maintenance',
+    'module_quality_mrp',
+]
+
 
 class ResConfigSettings(models.TransientModel):
     _inherit = 'res.config.settings'
 
+    @classmethod
```
Comment (yajo) [inline res_config_settings_enterprise_remove/models/res_config_settings.py]: `@staticmethod`

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 2]
```diff
@@ -1,14 +1,58 @@
...
+    def _format_query_by_field_name(self, field_name):
```
Comment (yajo) [inline res_config_settings_enterprise_remove/models/res_config_settings.py]: No `self`

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 3]
```diff
@@ -1,14 +1,58 @@
...
+        return "//div[div[field[@name='%s' and \
+                @widget='upgrade_boolean']]]" % field_name
```
Comment (yajo) [inline res_config_settings_enterprise_remove/models/res_config_settings.py]: Why not just search for the `@widget='upgrade_boolean'` part in loop until no more matches are found? 🤔 

That way we avoid maintaining the list of fields above...

**Review by yajo — CHANGES_REQUESTED** [review]

File: res_config_settings_enterprise_remove/models/res_config_settings.py (hunk 3 above)
Comment (astirpe) [reply res_config_settings_enterprise_remove/models/res_config_settings.py]: Thank you @Yajo ! Let's see how it behaves with that code simplification. I made a test and it seems working fine.

File: res_config_settings_enterprise_remove/models/res_config_settings.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+from lxml import etree
+
+from odoo import api, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    @staticmethod
+    def _format_query_by_field_name():
```
Comment (yajo) [inline res_config_settings_enterprise_remove/models/res_config_settings.py]: The method name now is misleading. Besides, I feel that we no longer need it...

**Review by yajo — CHANGES_REQUESTED** [review]

**Comment by yajo** [comment]
Functional test passed, code only 1 remark above.

**Comment by astirpe** [comment]
Squashed

File: res_config_settings_enterprise_remove/__manifest__.py [hunk 5]
```diff
@@ -1,18 +1,16 @@
...
 
 {
     'name': 'Settings - Remove Enterprise Fields',
-    'summary': 'Remove fields in all settings views marked as enterprise',
-    'version': '10.0.1.0.0',
+    'summary': 'Remove enterprise options from settings',
+    'version': '11.0.1.0.0',
     'category': 'Maintenance',
-    'website': "https://laslabs.com",
+    'website': 'https://github.com/OCA/server-tools',
```
Comment (yajo) [inline res_config_settings_enterprise_remove/__manifest__.py]: server-brand

**Review by yajo — APPROVED** [review]

**Review by feketemihai — APPROVED** [review]

**Pull request merged**